pylsp_mypy/plugin.py

```diff
@@ -95,6 +95,15 @@ def parse_line(line: str, document: Optional[Document] = None) -> Optional[Dict[
     return None
 
 
+def apply_overrides(args: List[str], overrides: List[Any]) -> Generator[str, None, None]:
```

Comment:
Why a generator if it gets casted to a list later anyway?

Reply:
Given that I just want to remove a single option I would have to specify a list of options except without the one I want to remove, with this way of "merging"?

Reply:
> Why a generator if it gets casted to a list later anyway?

Now returns a list instead.

> Given that I just want to remove a single option I would have to specify a list of options except without the one I want to remove, with this way of "merging"?

We can change to a DSL that offers a "set difference" operation. However, a good many (unary) options have inverse counterparts, which means they're undoable by appending negated variants . Please see the added note in the updated readme. Thanks.

Reply:
That's right, I did not think about that. Thank you. I guess it is fair to use this for now and if it should prove to be to limited in the future, this is something to be addressed in the future.